source/documentation/05-payment-overview.md

```diff
@@ -105,21 +108,57 @@ If the details are valid and the payment is approved, the user is then taken to
 
 ![](https://s3-eu-west-1.amazonaws.com/pay-govuk-documentation/flow-payment-confirm-page.png)
 
-After confirming, the user is directed to the ``return_url`` you provided in the initial request.
+The user clicks confirm, and what happens next depends on whether the payment was successful or not.
 
-If the payment *cannot* be approved, the user is shown an error page with a message describing the reason, for example:
+##### Successful Payment
 
-![](https://s3-eu-west-1.amazonaws.com/pay-govuk-documentation/flow-payment-declined.png)
+If the payment is successful, the user will receive a confirmation email (if you have chosen to send these using GOV.UK Pay), and return to your service and see your confirmation page.
+
+###### Confirmation Email
+
+The user will receive a payment confirmation email containing:
+
+- payment reference number
+- date of payment
+- who the payment was to
+- the total payment amount
+
+You can add a custom paragraph to a payment confirmation email at the [Email notifications page](https://selfservice.payments.service.gov.uk/email-notifications) on the GOV.UK Pay admin site. For further customisation, you can visit [GOV.UK Notify](https://www.notifications.service.gov.uk/) to set up custom notifications. It is recommended to disable GOV.UK Pay notifications if you do this.
+
+###### Confirmation Page
+
+The confirmation page is hosted by you and should:
+
+- confirm that the payment has been received successfully
+- contain a reference number (make them short and usable)
+- have a clear payment summary, showing the amount and description
+- clearly state what is going to happen next (this will be different for each service)
+- if applicable, let the user know they will receive a receipt email (services can either use GOV.UK Notify to send email payment receipts or ask GOV.UK Pay to do that for them)
+- direct the user to the ``return_url`` you provided in the initial request
 
-The link to try the payment again sends the user to the ``return_url`` you provided in the initial request.
+Users have different ways of recording this confirmation information, including screenshots, prints, pdf receipts to download, and writing down the reference number and other relevant information. Teams building services should be aware of, and cater for, all these behaviours.
+
+>Read more about confirmation pages in the [service manual](https://www.gov.uk/service-manual/design/confirmation-pages).
+
+##### Failed Payment
+
+If the payment fails - perhaps because it was declined or there was a technical error - the user will see a GOV.UK Pay error page. This includes a link to return to your service. When a user returns to your service after a failed payment you should show them a page that offers useful next steps. This page should:
+
+- confirm that the payment failed
+- offer the user a chance to try the payment again via the ``return_url`` link that you provided in the initial request
```

Comment:
I'm assuming that this isn't the latest version because it is still referencing the return_URL incorrectly

Reply:
You are correct, it isn't the latest version